src/components/SettlementButton.tsx

```diff
@@ -252,7 +274,7 @@ function SettlementButton({
 
     return (
         <KYCWall
-            onSuccessfulKYC={onPress}
+            onSuccessfulKYC={(paymentType) => onPress(paymentType)}
```

Comment:
```suggestion
            onSuccessfulKYC={onPress}
```
I think it will work right?